plugin/completion.py

```diff
@@ -224,6 +224,10 @@ def initialize(self) -> None:
                 'triggerCharacters') or []
             if trigger_chars:
                 self.register_trigger_chars(session, trigger_chars)
+            # This is to make ST match with labels that have a weird prefix like a space character.
+            # It is unfortunately untestable as this problem will only show up when typing characters, not when
```

Comment:
Just wanted to say that we already have a test for the case where the label contains spaces.

https://github.com/sublimelsp/LSP/blob/87e98cf9d0a7d80f315dfb74d22b6c568a2a34b3/tests/test_completion.py#L237-L245

If the label is `'override def myFunction(): Unit'`.

You can type `def myF`.

1. If you trigger AC and auto_complete_preserve_order is "some":
You wont see the AC, because filtered out that completion item.

2. If you trigger AC and auto_complete_preserve_order is "none",
You will see the AC

Here is an example for that.
https://github.com/sublimelsp/LSP/blob/87e98cf9d0a7d80f315dfb74d22b6c568a2a34b3/tests/test_completion.py#L260-L261

Just a reminder,
I set for all completion tests auto_complete_preserve_order to "none"
https://github.com/sublimelsp/LSP/blob/87e98cf9d0a7d80f315dfb74d22b6c568a2a34b3/tests/setup.py#L175

Reply:
Thanks, I had not noticed this was set up in the tests :)